src/base/registration/CheckNameAvailability.svelte

```diff
@@ -57,6 +57,13 @@
       error = err.message;
     }
   });
+
+  function goBack() {
```

Comment:
Sorry for commenting on your draft PR, but maybe we should find a uniform naming for going back in history.

Maybe we can rename `router.pop` into `router.back` since the router is essentially going back. Not necessary to let users know their popping items from the history stack. wdyt?

Reply:
> we're the users of the router API

True that, ok lets leave it